README.md

```diff
@@ -89,11 +89,12 @@ $ minikube addons disable metrics-server
 
 ### Kubernetes compatibility matrix
 
-| kube-prometheus stack | Kubernetes 1.14 | Kubernetes 1.15 | Kubernetes 1.16 | Kubernetes 1.17 |
-|-----------------------|-----------------|-----------------|-----------------|-----------------|
+| kube-prometheus stack | Kubernetes 1.14 | Kubernetes 1.15 | Kubernetes 1.16 | Kubernetes 1.17 | Kubernetes 1.18 |
+|-----------------------|-----------------|-----------------|-----------------|-----------------|-----------------|
 | `release-0.3`         | ✔              | ✔              | ✔              | ✔              |
 | `release-0.4`         | ✗              | ✗              | ✔              | ✔              |
-| `HEAD`                | ✗              | ✗              | ✗              | ✔              |
+| `release-0.5`         | ✗              | ✗              | ✗              | ✔              | ✔
```

Comment:
Any particular reason for the 1.16 (+downward) incompatibility? Mainly curious, I'm also happy with the answer being "let's just not care for more releases, and encourage people to upgrade".

Reply:
We do not test that in this branch, that is the only reason, we can't guarantee it, in the other branches we run our e2e tests against those k8s versions so we can say it passes.

At some point we have to decide how many versions we want to support.

Reply:
just wanted to make sure we're aligned, totally ok with this